zerotwobot/modules/connection.py

```diff
@@ -118,16 +116,19 @@ async def connect_chat(update: Update, context: ContextTypes.DEFAULT_TYPE):
                 await send_message(update.effective_message, "Invalid Chat ID!")
                 return
 
-            isadmin = getstatusadmin.status in ("administrator", "creator")
-            ismember = getstatusadmin.status in ("member")
+            isadmin = getstatusadmin.status in [ChatMemberAdministrator, ChatMemberStatus.OWNER]
```

Comment:
This should be ChatMemberStatus.ADMINISTRATOR

Reply:
Looks good now.